src/test/java/server/OwnerRepositoryTest.java

```diff
@@ -0,0 +1,35 @@
+package server;
+
+import entity.Owner;
+import org.junit.jupiter.api.Test;
+import server.OwnerRepository;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OwnerRepositoryTest {
+
+    private OwnerRepository ownerRepo;
+
+    @Test
+    void createOwner_OwnerExists() {
+        this.ownerRepo = new OwnerRepository();
```

Comment:
I changed the constructor of OwnerRepository to private, so you can no longer call that method.
Instead, you can use OwnerRepository.getOwnerRepository() to get a OwnerRepository object

Reply:
You will need to do 
    @Before
    public void setUp() {
}
to initialize things you need for every test
and 
@After
    public void tearDown() {
} 
to remove things you don't need for the next rounds of tests